docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -0,0 +1,308 @@
+	| master_addresses     | ['89.46.80.136']                                                                                        |
+	| master_lb_enabled    | False                                                                                                   |
+	| create_timeout       | 60                                                                                                      |
+	| node_count           | 1                                                                                                       |
+	| discovery_url        | https://discovery.etcd.io/23af721dc3ee773d2674db4881ff70cb                                              |
+	| docker_volume_size   | 50                                                                                                      |
+	| master_count         | 1                                                                                                       |
+	| container_version    | 1.12.6                                                                                                  |
+	| name                 | bangor                                                                                                  |
+	| master_flavor_id     | 2C-4GB-20GB                                                                                             |
+	| flavor_id            | 2C-4GB-20GB                                                                                             |
+	| health_status_reason | {'bangor-id6nijycp2wy-master-0.Ready': 'True', 'bangor-id6nijycp2wy-node-0.Ready': 'True', 'api': 'ok'} |
+	| project_id           | dfc700467396428bacba4376e72cc3e9                                                                        |
+	+----------------------+---------------------------------------------------------------------------------------------------------+
+	```
+
+## Accessing the Kubernetes cluster with kubectl
+
+You may install the Kubernetes command line tool, `kubectl`, on your 
+local computer, and run commands against your cluster. To install 
+`kubectl`, use the package manager of your operating system.
+
+=== "Debian/Ubuntu"
+	```bash
+	apt install kubectl
+	```
+=== "Mac OS X with Homebrew"
+	```bash
+	brew install kubectl
+	```
+
+Before running commands against a specific cluster, you must have the 
+corresponding `config` file on your computer.
+
+=== "{{gui}}"
+	Downloading a `config` file from the {{gui}} is currently not 
+	supported. You can still have the `config` file of your newly created 
+	Kubernetes cluster using the OpenStack CLI.
+=== "OpenStack CLI"
+
+	To download the `config` file for your Kubernetes cluster, type 
+	the following:
+
+	```bash
+	openstack coe cluster config bangor
+	```
+
+After saving the `config` file locally, set the value of variable 
+`KUBECONFIG` to the full path of the file. Type, for example:
+
+```bash
+export KUBECONFIG=${PWD}/config
```

Comment:
If you're using `${PWD}` in the `export` command below, you should also use `openstack coe cluster config --dir=${PWD} bangor`.

Reply:
Done!